src/Tizen.NUI/src/public/Color.cs

```diff
@@ -141,6 +141,28 @@ public Color(string hexColor) : this(Interop.Vector4.new_Vector4__SWIG_0(), true
             }
         }
 
+        /// <summary>
+        /// The conversion constructor from an System.Drawing.Color instance.
+        /// </summary>
+        /// <param name="color">System.Drawing.Color instance</param>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
```

Comment:
please change "tizen_6.0" as "tizen_next" if this API's open schedule is not confirmed.

Reply:
DONE